config.go

```diff
@@ -375,6 +377,115 @@ func (c *CascadingPaginationColumnConfig) FallbackPaginationColumnName() (string
 	return c.FallbackColumn, true
 }
 
+type DataIterationBatchSizePerTableOverride struct {
+	// Lower limit for rowSize, if a rowSize <= MinRowSize, ControlPoints[MinRowSize] will be used
+	MinRowSize int
+	// Upper limit for rowSize, if a rowSize >= MaxRowSize, ControlPoints[MaxRowSize] will be used
+	MaxRowSize int
```

Comment:
I'm fine with having this, but isn't this information kind of redundant? You can figure out a min and max by looking at the keys of the ControlPoints map? I suppose if you do it that way the documentation on ControlPoints will need to be a bit more complex.

Reply:
It is a bit redundant but I think this makes it more clear that we must provide a `Min` and `Max` RowSize. It also makes the code a little easier since we don't need to loop through and find the min and max